homework4/task2.js

```diff
@@ -0,0 +1,9 @@
+function plusFactory(num) {
+  return num;
+}
+const plusFive = (value) => plusFactory(value + 5);
```

Comment:
INCORRECT

u have to write code inside plusFactory but not change  plusFive and plusTen